packit/patches.py

```diff
@@ -456,6 +438,45 @@ def process_git_am_style_patches(
 
         return new_patch_list
 
+    @staticmethod
+    def check_patch_changes(
+        patch_list: List[PatchMetadata],
+        repo: git.Repo,
+    ) -> List[PatchMetadata]:
+        """
+        Check if the patches changed are indeed a change or can be reset.
```

Comment:
Seeing this the first time I have no idea how is the output list different from the input one (`patch_list`).
I guess the output is just somehow filtered input, but I don't want to read (if I was not reviewing this PR) the code to discover it.

Reply:
Renamed the method and improved the docstring. Let me know if still requires improvement.